src/kernel/kmain.zig

```diff
@@ -120,7 +122,6 @@ export fn kmain(boot_payload: arch.BootPayload) void {
         var ramdisk_filesystem = initrd.InitrdFS.init(&initrd_stream, &kernel_heap.allocator) catch |e| {
             panic_root.panic(@errorReturnTrace(), "Failed to initialise ramdisk: {}\n", .{e});
         };
-        defer ramdisk_filesystem.deinit();
```

Comment:
How come the `deinit` is removed?

Reply:
Since if the ramdisk was deinitialised then the scheduler wouldn't be able to read the user program from the vfs. I think we discussed privately about leaving the ramdisk in memory until we initialise a FAT system and change the root (at which point we won't need the ramdisk).